app/Presenters/Admin/DigitalPublicationArticlePresenter.php

```diff
@@ -74,4 +74,17 @@ public function citeAs()
 
         return $this->addCssClass($this->entity->cite_as, 'f-secondary');
     }
+
+    public function isArticleInTree($items, $currentArticle)
```

Comment:
Remove the `$currentArticle` parameter and use `$this->entity` in place of that variable.